src/frontend/apps/standalone_site/src/components/Styled/Link.tsx

```diff
@@ -0,0 +1,9 @@
+import { Link as LinkBase } from 'react-router-dom';
+import styled from 'styled-components';
+
+export const Link = styled(LinkBase)`
```

Comment:
There is an existing `StyledLink` component. They are different but can we mutualize them or not ?

Reply:
The hover that is quite different, I could put the hover as a props though. 

Something like that:
```
<StyledLink to="/uploads" onHover=`background: white;
    color: ${normalizeColor('brand', theme)};

    // Force the underline only on the link part, not on the count
    & .link-underlined-part {
      text-decoration: underline;
    }`>
```

Up to you

Reply:
Then can be mutualized and you should use your `StyledLink` to replace the existing one.